src/cosalib/qemuvariants.py

```diff
@@ -187,22 +200,26 @@ def mutate_image(self, callback=None):
             callback(work_img)
 
         if self.tar_members:
-            wmode = 'w'
-            if final_img.endswith('gz'):
-                wmode = 'w:gz'
-            elif final_img.endswith('xz'):
-                wmode = 'w:xz'
-            log.info(f"Preparing tarball with mode '{wmode}': {final_img}")
-            with tarfile.open(final_img, wmode) as tar:
-                base_disk = self.tar_members.pop(0)
-                base_name = os.path.basename(base_disk)
-                log.info(f" - adding base disk '{work_img}' as '{base_name}'")
-                tar.add(work_img, arcname=base_name)
-
-                for te in self.tar_members:
-                    te_name = os.path.basename(te)
-                    log.info(f" - adding additional file: {te_name}")
-                    tar.add(te, arcname=te_name)
+            # Python does not create sparse Tarfiles, so we have do it via
+            # the CLI here.
+            log.info(f"Preparing tarball {final_img} with flags '{self.tar_flags}'")
```

Comment:
Log seems unnecessary since we print the tar command?

Reply:
Given the level of logging that we already have, the extra logging seen innocuous. 

Removed.